CONTRIBUTING.md

```
@@ -57,30 +85,95 @@ Make sure you have read the [Documentation for Setting up the Project](https://g
+1. If you've added code, then test suites must be added.
+
+   1. **_General_:**
+
+      1. We need to get to 100% test coverage for the app. We periodically increase the desired test coverage for our pull requests to meet this goal.
+      1. Pull requests that don't meet the minimum test coverage levels will not be accepted. This may mean that you will have to create tests for code you did not write. You can decide which part of the code base needs additional tests if this happens to you.
+
+   2. **_Testing_:**
+
+      1. Test using this set of commands:
+
+         ```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)